### DISCUSSION TITLE
tailwind css beta 4

### DISCUSSION THREAD

**Discussion by sgcullen** [wxt-dev/wxt#1277] [General]
Has anyone had much luck getting tailwind css Beta 4 working
Theres are two install methods - 1 using a vite plugin and 1 using postcss:
https://tailwindcss.com/docs/v4-beta
There are a few nice new features that I'm keen to test out.

**Comment by aklinker1** [comment]
The vite plugin works just fine, for HTML pages and content scripts.

Here's an example that applies different styles for light/dark themes to a popup and content script UI.

[tailwind-v4-example.zip](https://github.com/user-attachments/files/18128321/tailwind-v4-example.zip)

https://github.com/user-attachments/assets/43873acf-7d6d-437d-a480-6199c715247e

**Reply by sgcullen** [reply]
Hey Aaron, thanks.
Yes - that looks to work fine in your example.
I'm using svelte at the moment with inline styles and that the tailwind classes in the .svelte files don't seem to get picked up automatically.

**Reply by aklinker1** [reply]
Yeah, not sure about that. Maybe head over to the tailwind repo and report an issue there?

**Reply by sgcullen** [reply]
Thanks Aaron, will do.
Thanks to you all for your help recently. Hopefully some of my recent questions have been useful for others too!
